README.md

```diff
@@ -2,21 +2,63 @@
 
-Risks outside the scope of this project:
+### Risks outside the scope of this project:
 
-- If a tenant controls its scrape target configuration the tenant can set arbitrary labels via its relabelling configuration, thereby being able to pollute other tenant's metrics.
+It's not a goal for this project to solve write tenant isolation for multi-tenant Prometheus:
 
-- If the ingestion configuration [honor_labels](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) is set for a tenant's target, that target can pollute other tenant's metrics as Prometheus respects any labels exposed by the target.
+* If a tenant controls its scrape target configuration the tenant can set arbitrary labels via its relabelling configuration, thereby being able to pollute other tenant's metrics.
+* If the ingestion configuration [honor_labels](https://prometheus.io/docs/prometheus/latest/configuration/configuration/#scrape_config) is set for a tenant's target, that target can pollute other tenant's metrics as Prometheus respects any labels exposed by the target.
+
+See [Thanos soft/hard tenancy](https://thanos.io/tip/proposals/201812_thanos-remote-receive.md/#architecture) or [Cortex](https://cortexmetrics.io/) as example solution to that.
+
+## Installing `prom-label-proxy`:
+
+### Docker 
+
+We publish docker images for each release, see:
+
+* [`quay.io/prometheuscommunity/prom-label-proxy`](https://quay.io/repository/prometheuscommunity/prom-label-proxy?tab=tags) for newest images
+* `quay.io/coreos/prom-label-proxy:v0.1.0` for the initial v0.1.0 release.
+
+### Building from source
+
+If you want to build `prom-label-proxy` from source you would need a working installation of
+the [Go](https://golang.org/) 1.15+ [toolchain](https://github.com/golang/tools) (`GOPATH`, `PATH=${GOPATH}/bin:${PATH}`).
+
+`prom-label-proxy` can be downloaded and built by running:
+
+```bash
+go get github.com/prometheus-community/prom-label-proxy
+```
 
 ## How does this project work?
 
 This application proxies the `/federate`, `/api/v1/query`, `/api/v1/query_range`, `/api/v1/rules`, `/api/v1/alerts` Prometheus endpoints as well as `/api/v2/silences` Alertmanager endpoint and it ensures that a particular label is enforced in the particular request and response.
 
-Once again for clarity: this project only enforces a particular label in the respective calls to Prometheus, it in itself does not authenticate or authorize the requesting entity in any way, this has to be built around this project.
+Particularly, you can run `prom-label-proxy` with label `tenant` and point to example Prometheus server e.g:
+
+```
+prom-label-proxy \
+   -label tenant \
+   -upstream http://demo.robustperception.io:9090 \
```

Comment:
nice touch :wink:

Reply:
@brian-brazil would approve (:

Reply:
prometheus org has a demo site exactly for this purpose: https://prometheus.demo.do.prometheus.io/graph